TMGToolbox/src/network_editing/network_comparison/copy_transit_lines.py

```diff
@@ -107,6 +111,9 @@ def __init__(self):
         self.SourceEmmebankPath = _MODELLER.emmebank.path
         self.SourceScenarioId = _MODELLER.scenario.id
         self.TargetScenario = _MODELLER.scenario
+        self.LinksAllowedForModification = None
+        self.linkModAttrID = '@link_added_mode'
```

Comment:
Instead of using a fixed name for the attribute, we should instead use a temporary attribute from the "tempExtraAttributeMANAGER" in "common.utilities".